src/support_function.jl

```diff
@@ -1,25 +1,25 @@
-# =======================================
-# Support function of an abstract LazySet
-# =======================================
+# =============================
+# Support function of a LazySet
+# =============================
 
 """
-    ρ(d::Vector{Float64}, sf::LazySet)::Float64
+    ρ(d::Vector{N}, S::LazySet)::N where {N<:Real}
 
 Evaluate the support function of a set in a given direction.
 
 ### Input
 
-- `d`  -- a real vector, the direction investigated
-- `sf` -- a convex set
+- `d` -- a real vector, the direction investigated
+- `S` -- a convex set
 
 ### Output
 
-- `ρ(d, sf)` -- the support function
+The support function of the set `S` for the direction `d`.
 """
-function ρ(d::Vector{Float64}, sf::LazySet)::Float64
-    return dot(d, σ(d, sf))
+function ρ(d::Vector{N}, S::LazySet)::N where {N<:Real}
+    return dot(d, σ(d, S))
 end
 
-# alias
+# aliases
 support_function = ρ
```

Comment:
let's comment them, such as:

```
"""
    support_function

Alias for the support function ρ.
"""
```

and 

```
"""
    support_vector

Alias for the support vector σ.
"""
```
respectively.

Reply:
done